template-helpers/breach-detail.js

```diff
@@ -5,6 +5,27 @@ const { prettyDate } = require("./hbs-helpers");
 const { getAllPriorityDataClasses, getAllGenericRecommendations, getFourthPasswordRecommendation } = require("./recommendations");
 
 
+function addUtmParams(cta) {
```

Comment:
nit: more explicit function name would be `addContextualRecommendationUtmParams` since the param values here are hard-coded to `utm_campaign: "contextual-recommendations"`